data/json/items/tool_armor.json

```
@@ -390,7 +392,13 @@
     "qualities": [ [ "GLARE", 1 ] ],
     "flags": [ "SUN_GLASSES", "FLASH_PROTECTION" ],
     "armor": [
-      { "encumbrance": 20, "coverage": 90, "covers": [ "head" ] },
+      {
+        "encumbrance": 20,
+        "coverage": 100,
+        "covers": [ "head" ],
+        "specifically_covers": [ "head_forehead", "head_ear_r", "head_ear_l" ]
+      },
+      { "encumbrance": 20, "coverage": 30, "covers": [ "head" ], "specifically_covers": [ "head_crown" ] },
```

```suggestion
      { "encumbrance": 0, "coverage": 30, "covers": [ "head" ], "specifically_covers": [ "head_crown" ] },
```

Otherwise total encumbrance on head is 40